### PR TITLE
[#51] updating styless.css to fix image alignment issues

### PR DESCRIPTION
**Pull request by shailee-m**
## Fixes
- Fixes https://github.com/creativecommons/project_creativecommons.org/issues/51 by @brylie 

## Description
Removed display CSS from alignment class.

## Checklist
- [x] My pull request has a descriptive title (not a vague title like `Update
  index.md`).
- [x] My pull request targets the *default* branch of the repository (`main` or `master`).
- [x] My commit messages follow [best practices][best_practices].
- [x] My code follows the established code style of the repository.
- [ ] I added or updated tests for the changes I made (if applicable).
- [ ] I added or updated documentation (if applicable).
- [ ] I tried running the project locally and verified that there are no
  visible errors.

[best_practices]:https://gist.github.com/robertpainsi/b632364184e70900af4ab688decf6f53

## Developer Certificate of Origin

For the purposes of this DCO, "license" is equivalent to "license or public domain dedication," and "open source license" is equivalent to "open content license or public domain dedication."
<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>

### CODE REVIEW

**Comment by shailee-m** [comment]
cc: @brylie

File: assets/css/styles.css [hunk 1]
```diff
@@ -1,3 +1,3 @@
...
...fset-11{grid-column:span 12}}@media screen and (max-width: 768px){.main-content .container{padding-left:1rem;padding-right:1rem}}@media screen and (max-width: 768px){.main-content .container.mobile-no-padding{padding-left:0;padding-right:0}}.page .main-content,.single .main-content{padding-top:0;padding-bottom:2rem}.page .main-content .entry-page-content,.single .main-content .entry-page-content{padding-top:2.5rem;padding-bottom:3rem}.page-template-page-home .main-content{padding-top:0;padding-bottom:0}.page-template-page-home .homepage-sidebars{margin-top:3rem}.main-header .navbar .navbar-menu .menu-item.current-menu-item{color:#000}.main-header .navbar .navbar-menu .menu-item.current-menu-item>a{color:#000}.main-header .navbar .navbar-menu .menu-item.current-menu-ancestor>a{color:#000}.push-up{margin-top:-4rem}.terms-list h6{margin-bottom:.5rem}.terms-list .button{margin-right:1rem;margin-bottom:.5rem}.error404 .page-header .container{padding-top:10rem;padding-bottom:10rem}.error404 .page-header .container h4{margin-bottom:1rem}.text-format p{margin-bottom:1.5rem}.text-format ul{padding-left:1.5rem;list-style:disc}.text-format ul li{margin-bottom:1rem}.text-format ol{list-style:n
```
Comment (brylie) [inline assets/css/styles.css:2]: I'm in favor of removing this mini reset line altogether.

**Review by brylie — CHANGES_REQUESTED** [review]

File: assets/css/styles.css (hunk 1 above)
Comment (shailee-m) [reply assets/css/styles.css:2]: it is a comment for the styles that follow. removing it breaks the display

Comment (brylie) [reply assets/css/styles.css:2]: Yes, it seems to be the minified Vocabulary styles inlined into this file. We need to figure out how to develop the Vocabulary styles in a single location.

**Pull request closed without merging**